docs/swagger-ui-http4s/src/main/scala/sttp/tapir/swagger/http4s/SwaggerHttp4s.scala

```diff
@@ -24,11 +24,11 @@ import scala.concurrent.ExecutionContext
   */
 class SwaggerHttp4s(
     yaml: String,
-    contextPath: String = "docs",
+    contextPath: List[String] = List("docs"),
```

Comment:
To have binary compatibility the function can take a `String` and the `rootPath` would be `Path(contextPath)` instead of the current implementation.

Reply:
This will be part of 0.18 so binary compat is not a must